cmd/lambda/leases/main.go

```diff
@@ -31,21 +45,33 @@ const (
 	Weekly               = "WEEKLY"
 )
 
-var muxLambda *gorillamux.GorillaMuxAdapter
+var (
+	muxLambda *gorillamux.GorillaMuxAdapter
+	//CurrentAccountID is the ID where the request is being created
+	// Services handles the configuration of the AWS services
+	Services *config.ServiceBuilder
+	// Settings - the configuration settings for the controller
+	Settings *leaseControllerConfiguration
+)
 
 var (
-	config                   common.DefaultEnvConfig
-	awsSession               *session.Session
-	dao                      db.DBer
-	snsSvc                   common.Notificationer
-	usageSvc                 usage.Service
-	leaseAddedTopicARN       *string
-	principalBudgetAmount    *float64
-	principalBudgetPeriod    *string
-	maxLeaseBudgetAmount     *float64
-	maxLeasePeriod           *int64
-	defaultLeaseLengthInDays *int
+	// Soon to be deprecated - Legacy support
+	Config             common.DefaultEnvConfig
+	awsSession         *session.Session
+	dao                db.DBer
+	snsSvc             common.Notificationer
+	usageSvc           usage.Service
+	leaseAddedTopicARN string
```

Comment:
Aren't some of these also in leaseControllerConfiguration?

Reply:
For now we are supporting both old and new ways of interacting with db layer.